docs/ref/models/querysets.txt

```diff
@@ -2221,7 +2221,7 @@ normally supports it).
 .. method:: bulk_update(objs, fields, batch_size=None)
 
 This method efficiently updates the given fields on the provided model
-instances, generally with one query::
+instances, generally with one query, and returns the count of objects updated::
```

Comment:
Use `number` over `count` for consistency with the same change requested elsewhere.

```suggestion
instances, generally with one query, and returns the number of objects updated::
```

Reply:
this isn't necessarily part of this patch. shouldn't it ideally require a separate commit?

Reply:
sorry for the noise, it is part of the patch. updated this as per the suggestion.